src/en/howto-privatecloud.md

```diff
@@ -230,36 +255,47 @@ The output from the previous command will be similar to the following:
 The URL for the Object Store is listed against the `internalurl` field above
 and we refer to this as **$SWIFT_URL** in the following commands.
 
+!!! Note: you can verify the url before bootstrap with
+wget $SWIFT_URL/simplestreams/images/streams/v1/index.json
+
 Enter the following command to register the endpoint with the Simplestreams
 service:
 
 ```bash
-openstack endpoint create --region $REGION --publicurl $SWIFT_URL/simplestreams/images \
+openstack endpoint create --region $REGION \
+   --publicurl $SWIFT_URL/simplestreams/images \
    --internalurl $SWIFT_URL/simplestreams/images product-streams
 ```
 
-### Bootstrap with Juju
+!!! Note: juju will automatically look for a product-streams service during
+bootstrap to use for image streams.
 
-Now the Simplestream service is registered and running you can create a controller on
-this cloud with the `juju bootstrap` command. 
+## Using the Glance Simplestreams Sync charm to configure image streams.
 
-```bash
-juju bootstrap <cloud> <controller name> --config image-metadata-url=$SWIFT_URL
-```
+The Glance Simplestreams Sync charm will do all of the above work for you and
+provide customizeable syncing for automatic image updates.
 
-If there are multiple possible networks available to the cloud, it is also necessary to 
-specify the network label or UUID for Juju to use. Both the network label and
-UUID can be retrieved with the following command:
+### Requirements
 
-```bash
-openstack network list
-```
+ - OpenStack deployment by Juju
 
-Finally, use either the network label or the UUID with the 'network' configuration
-option when bootstrapping a new controller:
+!!! Note: You must have permissions to deploy charms in the juju model running
+OpenStack to utilize this method for image metatdata management.
 
 
-```bash 
-juju bootstrap openstack --config image-metadata-url=$SWIFT_URL --config network=<network id>
-```
+### Deploying the charm to your OpenStack Cloud
```

Comment:
Could we change "the charm" to something more specific like "the Glance simple streams charm" or "the metadata management charm". I'm just worried about the distinction between deploying charms using your new mystack cloud versus deploying charms to the underlying infrastructure and I think the title "Deploying the charm to your OpenStack Cloud" could refer to either.

Reply:
updated.